client/sites/components/site-preview-pane/constants.ts

```diff
@@ -9,7 +9,6 @@ export const DOTCOM_STAGING_SITE = 'dotcom-staging-site';
 export const DOTCOM_SITE_PERFORMANCE = 'dotcom-site-performance';
 
 export const SITE_MARKETING_TOOLS = 'site-marketing-tools';
-export const SITE_MARKETING_BUSINESS_TOOLS = 'site-marketing-business-tools';
```

Comment:
Can we exclude any changes under `client/sites/*` and `client/sites/marketing/*`; it's under rapid development and will clash with some of my next PRs, it would be great to avoid it now 🙏 🙏  (it's still just stub anyway, I'll take care of remove the stub biz tools tab in my PR)

Reply:
Sure, done!